app/models/child_benefit_tax_calculator.rb

```diff
@@ -46,7 +48,11 @@ def nothing_owed?
   end
 
   def has_errors?
-    errors.any? || starting_children.select { |c| c.errors.any? }.any?
+    errors.any? || starting_children_errors?
+  end
+
+  def starting_children_errors?
```

Comment:
Not sure about this. If `is_part_year_claim=='no'` then there are no errors?. I might be missing something. My understanding is that we would need to validate all the children regardless the `is_part_year_claim` value.